encoding/rs/serialization.go

```diff
@@ -0,0 +1,43 @@
+package rs
+
+import (
+	"errors"
+
+	"github.com/consensys/gnark-crypto/ecc/bn254/fr"
+)
+
+const BYTES_PER_SYMBOL = 32 // fr.Element serializes to 32 bytes
```

Comment:
We can reuse already defined constant: https://github.com/Layr-Labs/eigenda/blob/82802dbcf1cf929781c6249ab476f4f03769a532/encoding/constants.go#L9

Reply:
Sorry meant to delete this file, it's not relevant